plugins/net_tools.go

```diff
@@ -0,0 +1,62 @@
+package plugins
+
+import (
+	"net"
+	"os/exec"
+	"strings"
+
+	"github.com/belak/irc"
+	"github.com/belak/seabird/bot"
+	"github.com/belak/seabird/mux"
+)
+
+func init() {
+	bot.RegisterPlugin("net_tools", NewNetToolsPlugin)
+}
+
+func NewNetToolsPlugin(m *mux.CommandMux) error {
+	m.Event("dig", Dig)
+	m.Event("ping", Ping)
+	m.Event("dnscheck", DnsCheck)
+
+	return nil
+}
+
+func Dig(c *irc.Client, e *irc.Event) {
+	if e.Trailing() == "" {
+		c.MentionReply(e, "Domain required")
+		return
+	}
+
+	addrs, err := net.LookupHost(e.Trailing())
+	if err != nil {
+		c.MentionReply(e, err.Error())
+	}
+
+	c.MentionReply(e, addrs[0])
```

Comment:
This line could panic if addrs is nil, which would happen for sure if `err != nil`. Perhaps checking the len of addrs and giving them info on that would be good, especially in case someone looks something up with no records.